app/components/aircraft-queue.js

```diff
@@ -0,0 +1,82 @@
+import Ember from 'ember';
+const {computed} = Ember;
+import {TYPES as AIRCRAFT_TYPES, SIZES as AIRCRAFT_SIZES} from '../models/aircraft';
+
+// poor man's method of generating unique numbers,
+// purely to assist human readers in identifying airplanes
+const numbers = [];
+function makeNumber() {
```

Comment:
I found [this](https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto) the other day and figured I'd share.

So your ```makeNumber``` function would be simplied to:

```
function makeNumber() {
  return window.crypto.getRandomValues(new Uint32Array(1))[0];
}
```

Not a game changer, just a neat trick I learned a few days ago.